pages/invitation/InvitationHandler.php

```diff
@@ -229,6 +229,92 @@ public function invite(array $args, Request $request): void
         $templateMgr->display('/invitation/userInvitation.tpl');
     }
 
+    /**
+     * Edit user using user access table action
+     * @param $args
+     * @param $request
+     * @return void
+     * @throws \Exception
+     */
+    public function editUser($args, $request): void
+    {
+        $invitation = null;
+        $invitationPayload =[];
```

Comment:
I am wondering if you can use the already existing `UserRoleAssignmentInvitePayload` object here, in order to avoid the many key strings. 

What do you think?